pytition/petition/templates/petition/transfer_petition.html

```diff
@@ -0,0 +1,117 @@
+{% extends "layouts/layout.html" %}
+{% load i18n %}
+{% block container_type %}container{% endblock container_type %}
+
+{% block main_content %}
+<div class="container">
+<div class="row">
+      <div class="col-12">
+          <h2>{% trans "Transfer petition" %}</h2>
+      </div>
+</div>
+<div class="row">
+      <div class="col-12">
+          <div class="alert alert-warning">
+              {% blocktrans %}
+              You are going to transfer this petition to someone else:
+              {% endblocktrans %}
+              "<i>{{ petition.title }}</i>"
+          </div>
+      </div>
+</div>
+  <div class="row">
+      <div class="col-6">
+          <form>
+              <div class="form-group">
+                  <label for="searchbox">{% trans "Enter the name of the user or organization you want to give the petition to:" %}</label>
+                <input type="text" id="searchbox" placeholder="{% trans "User or Organization name" %}">
+              </div>
+          </form>
+          <ul class="list-group" id="search_results">
+          </ul>
+      </div>
+  </div>
+</div>
+
+  <div class="modal fade" id="modal_xfer_confirm" data-petition="" data-petition-delete="{% url 'petition_delete' petition.id %}">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h4 class="modal-title">{% trans "Transfer this petition?" %}</h4>
+          <button type="button" class="close" data-dismiss="modal">&times;</button>
+        </div>
+
+        <div class="modal-body">
+          <b>
+          {% blocktrans %}
+            Do you really want to transfer the following petition?
+          {% endblocktrans %}
+          </b>
+            <br/>
+          <i>'{{ petition.title }}'</i>
+          <br/>
+          <b>
+          {% blocktrans %}
+              It will be transfered to:
+          {% endblocktrans %}
+          </b>
+           <br/>
+           <i><div id="xfer_to_name"></div></i>
+        </div>
+
+        <div class="modal-footer">
+          <button type="button" class="btn btn-danger" data-action="petition-transfer" data-dismiss="modal">{% trans "Transfer" %}</button>
+          <button type="button" class="btn btn-info" data-dismiss="modal">{% trans "Cancel" %}</button>
```

Comment:
The idea was that "transfering" is kind of the "dangerous" option, kind of like a "delete" would be.
Since there is no way back. That's why I used red for the "Transfer". What do you think?

Reply:
That's what I thought.
I just read a bit about this topic and found this interesting thread: https://ux.stackexchange.com/questions/49991/should-yes-delete-it-be-red-or-green

So I guess you're right :). Maybe changing the "Cancel" button from `info` to `secondary`?

Reply:
Ok for `secondary` :)